src/vtquery.cpp

```diff
@@ -308,6 +388,10 @@ struct Worker : Nan::AsyncWorker {
                     while (auto feature = layer.next_feature()) {
                         auto original_geometry_type = get_geometry_type(feature);
 
+                        if (filter_enabled && filter_feature(feature, filters)) {
```

Comment:
If we have filters and the filters say we should filter out this feature, skip to next feature

Reply:
Helpful! 👍 Mind adding this as an inline comment?